### DISCUSSION TITLE
哪怕被人喷剧透我也认了！！！！！

### DISCUSSION THREAD

**Discussion by The-Place-Where-Winter-Sieeps** [share121/inter-knot#4735] [其他]
![屏幕截图(147)](https://github.com/user-attachments/assets/c80aeab9-a944-4ff7-8cb7-6b7b8aab32f0)
千万不要去选择求教！！！！！
你求救那个那个救会死！！！！
我是真后悔了！！！！
我服了😭😭😭

**Comment by keyuanya** [comment]
你也退出重进游戏，结果发现无法使用世界之外的力量把他们救回来😫

**Reply by The-Place-Where-Winter-Sieeps** [reply]
对！！！

**Comment by lllliufeng** [comment]
后面步离人会告诉你你找谁谁似

**Reply by The-Place-Where-Winter-Sieeps** [reply]
我还以为最多把椒丘打一顿不会杀死。
谁能想到会死别人啊！！！
<img src="https://img-static.mihoyo.com/communityweb/upload/7bd889870907d2d96673d09afb53cf63.png" alt="三月七_哭" width="75" >
